TestModels/Dependencies/Makefile

```diff
@@ -45,10 +45,10 @@ SMITHY_DEPS=dafny-dependencies/Model/traits.smithy
 GO_MODULE_NAME="github.com/Smithy-dafny/TestModels/Dependencies"
 
 GO_DEPENDENCY_MODULE_NAMES := \
-	--dependency-module-name=simple.constraints=github.com/ShubhamChaturvedi7/Constraints \
-	--dependency-module-name=simple.errors=github.com/ShubhamChaturvedi7/Errors \
-	--dependency-module-name=simple.extendable.resources=github.com/ShubhamChaturvedi7/Extendable \
-	--dependency-module-name=simple.resources=github.com/ShubhamChaturvedi7/Resource
+	--dependency-module-name=simple.constraints=github.com/Smithy-dafny/TestModels/Constraints \
```

Comment:
Great work on fixing this - maybe we should also fix `Smithy-dafny` to `smithy-dafny`.

Reply:
The problem is every test model has `Smithy-dafny`. It would be a huge fix in every test model. Its good to have this but its also fine to have `smithy-dafny`

Reply:
I have put a comment to track this in cleanup sim and let's do it when any one of us gets some free time.